src/Propel/Generator/Command/Console/Input/ArrayInput.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
```

Comment:
I directly copied this file sf.

Symfony fixed a bug that PHP7.4 has but it wasn't ported to 4.0, 4.1, 4.2 or 4.3. We also support those versions right now.